pkgs/applications/editors/typora/default.nix

```diff
@@ -15,16 +15,12 @@ stdenv.mkDerivation rec {
 
   unpackPhase = "dpkg-deb -x $src .";
 
-  dontWrapGApps = true;
```

Comment:
Not disabling this will produce two wrappers, that's why I inject `gappsWrappersArgs[@]` below.

However I was doing the wrapping in the wrong phase, and should be in `postFixup`.
Think that will fix the crash.